athoslag/MovieDB
# [Dev] Implementation for the App, as requested

As you can verify in the files and read in the updated README.md, I've implemented the application accordingly to the requested constraints. 
For more detail, see README.md.

What to test:
- Code review;
- App navigation flow;
- Load (and verify) more than 20 movies in Upcoming section (scroll past current collection middle)
- Select and view a movie details
- Switch to Search tab, make a search request (only requests when 'search' button is pressed)
- Select a movie from search, view its details
- (Optional) Make a second search and verify the previous data is removed
